Apps/WebClient/src/ClientApp/src/components/timeline/linearTimeline.vue

```diff
@@ -87,6 +99,32 @@
                 />
             </b-row>
         </div>
+        <div
+            v-if="filteredTimelineEntries.length == 0"
```

Comment:
nit: Having a get method that gives if there are items visible would be cleaner like
```
get timelineEmpty () : bool {
   return   filteredTimelineEntries.length == 0;
}
```

Reply:
good point!